### ISSUE TITLE
Regression: emsdk install llvm-git-master-64bit no longer works

### ISSUE THREAD

**Issue by juj** [emscripten-core/emsdk#573]
```
./emsdk install llvm-git-master-64bit
```
is expected to install Wasm backend LLVM from upstream git repo via git clone and build. If it was already installed, it should git pull latest and rebuild. Today after updating emsdk to latest, it no longer does so, but instead prints out

```
Skipped installing llvm-git-master-64bit, already installed.
```

i.e. the incremental update path has regressed. Before my update, it looks like I had emsdk from May 8th on this test system, where the incremental install did still work.

**Comment by sbc100** [comment]
Yeah I think that was probably #460..  I guess we need to avoid that for git based installs.

**Comment by sbc100** [comment]
Fixed in #647
